serverless.yml

```diff
@@ -135,6 +140,13 @@ functions:
           path: userhacks/{id}
           method: get
           cors: true
+  upload:
+    handler: ./services/hack/uploadSubmission.upload
+    events:
+      - http:
+          path: upload
+          method: post
+          cors: true
```

Comment:
Were going to want authentication on this endpoint to stop just anyone creating upload lingsuploading files and filling up out S3 buckets

Reply:
Yep, had it without it for testing purposes, it is now protected by authentication